include/deal.II/fe/fe_nothing.h

```diff
@@ -91,6 +91,13 @@ class FE_Nothing : public FiniteElement<dim, spacedim>
    */
   FE_Nothing(const unsigned int n_components = 1, const bool dominate = false);
 
+  /**
+   * Same as above but with the possibility to specify a reference-cell type.
```

Comment:
You don't have the possibility to specify a reference-cell type, you have to specify it. Right?